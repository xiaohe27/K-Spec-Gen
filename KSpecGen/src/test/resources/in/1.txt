class sum {

    static int sum_iterative(int n) {
        int s;
        s = 0;
        while (n > 0)
        {
            s = s + n;
            n = n - 1;
        }
        return s;
    }
}


