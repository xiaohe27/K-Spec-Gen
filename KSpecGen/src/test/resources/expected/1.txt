 while ( cast ( bool , cast ( int , 'ExprName(String2Id("n")) ) > 0 :: int ) ) {
                  cast ( int , ('ExprName(String2Id("s")) = cast ( int , cast ( int , 'ExprName(String2Id("s")) ) + cast (
                  int , 'ExprName(String2Id("n")) ) ))::AssignExp ) ; cast ( int , ('ExprName(String2Id("n")) = cast ( int , cast ( int ,
                   'ExprName(String2Id("n")) ) - 1 :: int ))::AssignExp ) ; }